src/screens/AdminPortal/OrganizationFunds/modal/FundModal.tsx

```
@@ -182,103 +189,103 @@ const FundModal: React.FC<InterfaceFundModal> = ({
+              checked={formState.isDefault}
+              onChange={() =>
+                setFormState({
+                  ...formState,
+                  isDefault: !formState.isDefault,
+                })
+              }
+            />
+          </Form.Group>
+          {mode === 'edit' && (
+            <Form.Group className="d-flex">
+              <label>{t('archived')} </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)